app/views/users/admin/stats.scala.html

```diff
@@ -0,0 +1,128 @@
+                        <h4 class="panel-title pull-left">Flags</h4>
+                        <div class="clearfix"></div>
+                    </div>
+                    <canvas id="chart-flags" height="40vh" width="100%"></canvas>
+                </div>
+            </div>
+        </div>
+    </div>
+    <script nonce="@nonce" src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.1/Chart.min.js"></script>
+    <script nonce="@nonce">
+            $(function(){
+                var domChartReview = document.getElementById("chart-reviews");
+                var chartReviews = new Chart(domChartReview, {
+                    responsive: true,
+                    type: 'line',
+                    data: {
+                        labels: @Html(Json.stringify(Json.toJson(reviews.map(_._2).toList))),
+                        datasets: [{
+                            label: "Reviews",
+                            backgroundColor: "cornflowerblue",
+                            borderColor: "dodgerblue",
+                            fill: false,
+                            data: @Html(Json.stringify(Json.toJson(reviews.map(_._1).toList)))
+                        }, {
+                            label: "Uploads",
+                            backgroundColor: "lightseagreen",
+                            borderColor: "darkseagreen",
+                            fill: false,
+                            data: @Html(Json.stringify(Json.toJson(uploads.map(_._1).toList)))
+                        }]
+                    },
+                    options: {
+                        title: {
+                            text: "Reviews"
+                        }
+                    }
+                });
+
+                var domChartDownload = document.getElementById("chart-downloads");
+                var chartDownloads = new Chart(domChartDownload, {
+                    responsive: true,
+                    type: 'line',
+                    data: {
+                        labels: @Html(Json.stringify(Json.toJson(totalDownloads.map(_._2).toList))),
+                        datasets: [{
+                            label: "Safe Downloads",
+                            backgroundColor: "cornflowerblue",
+                            borderColor: "dodgerblue",
+                            fill: false,
+                            data: @Html(Json.stringify(Json.toJson(totalDownloads.map(_._1).toList)))
+                        }, {
+                            label: "Unsafe Downloads",
+                            backgroundColor: "lightseagreen",
+                            borderColor: "darkseagreen",
+                            fill: false,
+                            data: @Html(Json.stringify(Json.toJson(unsafeDownloads.map(_._1).toList)))
+                        }]
+                    },
+                    options: {
+                        title: {
+                            text: "Reviews"
```

Comment:
This should be `Downloads`

Reply:
This is fixed now but you placed your comment on the wrong line :)